### ISSUE TITLE
Are Shearmaps being plotted correctly?

### ISSUE THREAD

**Issue by sweverett** [drphilmarshall/Pangloss#38]
The overlay plots of the convergence and shear maps look to be aligned, but the shear sticks don't appear to be plotting in the correct orientation. Here is an example image:
![gamma_normal](https://cloud.githubusercontent.com/assets/8742179/8445571/c3926216-1f53-11e5-83c2-f42fb9652122.png)
One possible fix could be that the .gamma_1 and .gamma_2 files are in the opposite order of how I am using them (i.e. I'm using $\gamma=\gamma_1+i\gamma2$ when instead it is $\gamma=\gamma_2+i\gamma_1$). When I made this change I got the following image:
![gamma_flipped](https://cloud.githubusercontent.com/assets/8742179/8445606/62f84a1e-1f54-11e5-909a-2cd5f38ca1ca.png)
Neither images are convincing. We should develop a test that doesn't rely on eyeballing the images to make sure we are plotting the shear sticks correctly.

**Comment by sweverett** [comment]
@drphilmarshall: I found an alternate definition of the shear for a polar coordinate frame centred at a cluster (which is what I'm pretty sure we're doing) at the bottom of [this page](http://gravitationallensing.pbworks.com/w/page/15553259/Weak%20Lensing#TangentialShear). I made the change to Shearmaps and got the following image:

![gamma_neg](https://cloud.githubusercontent.com/assets/8742179/8447047/ab655964-1f67-11e5-920c-c33a099f09d7.png)

Clearly it's not completely correct, but the top and bottom of the cluster look much more promising. Let me know what you think.
